recipes/muparser/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+sh ./configure --prefix=$PREFIX --disable-samples
+make -j2
```

Comment:
We have tried to remove these because of the CI issues that @msarahan has cited above. To be clear, I think you are right to raise this point.

That being said, if there is some amount (preferably the `floor` of the number of CPUs) that is ok, it would be nice to understand what that is and if it is greater than 1.